### PR TITLE
Add meetups for Aug, Oct, Nov, Dec

### PR DESCRIPTION
**Pull request by asgrim**

### CODE REVIEW

File: data/meetups/2015-11-11.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+END;
+
+$meetup->setId(0)
+    ->setFromDate(new DateTime('2015-11-11 19:00'))
+    ->setToDate(new DateTime('2015-11-11 23:00'))
+    ->setRegistrationUrl("http://www.eventbrite.co.uk/event/{$eid}")
+    ->setLocationUrl("https://www.google.co.uk/maps?q=Oasis+Venue,+Arundel+Street,+PO1+1NP&hl=en&ll=50.799642,-1.086724&spn=0.011772,0.031629&sll=50.799734,-1.086874&sspn=0.011772,0.031629&hq=Oasis+Venue,&hnear=Arundel+St,+PO1+1NP,+United+Kingdom&t=m&z=16")
+    ->setLocation('Oasis the Venue, Arundel Street, PO1 1NP')
+    ->setTalkingPoints(array(
+    	new TalkEntity('Phil Bennett and Jackson Willis', 'phil_bennett', 'tbc', $abstract),
```
Comment (lboynton) [inline data/meetups/2015-11-11.php]: Any way we can have both twitter handles here?

Comment (asgrim) [inline data/meetups/2015-11-11.php]: I don't think so, without correcting the model to correctly represent a speaker...

**Comment by lboynton** [comment]
Looks good to me, shall I merge?

**Comment by asgrim** [comment]
Sure :)

**Comment by richardjh** [comment]
asgrim lboynton Why is "Rob Allen" down as "Rob Allen / akrabat" ?

**Pull request merged**